app/gui2/src/main.ts

```diff
@@ -5,6 +5,7 @@ const INITIAL_URL_KEY = `Enso-initial-url`
 import './assets/main.css'
 
 import { basicSetup } from 'codemirror'
+import * as dashboard from 'enso-authentication'
```

Comment:
I believe that the side-effect of importing this statically will be that the dashboard code will no longer be split into its own chunk. That's fine, but it will likely cause vite to emit a "large chunk size" warning, which gui2 can't do much about. I guess we could try to improve the overall bundle size later.